.github/workflows/test.yml

```diff
@@ -270,13 +271,21 @@ jobs:
     - uses: actions/checkout@v3
       name: Checkout TTK source code
 
+    - name: Remove hosted Python
+      run: |
+        sudo rm -rf /Library/Frameworks/Python.framework
+        sudo rm -rf /usr/local/Frameworks/Python.framework
+
+
     - name: Install macOS dependencies
       run: |
         # ParaView dependencies
+        brew update
```

Comment:
the current version of the repo does not include this line

Reply:
It was before I made the other PR so that the tests pass.
I will remove the commit later so that it does not break your local branch :)

Reply:
Removed!